moderation/topwarnings.go

```diff
@@ -0,0 +1,70 @@
+package moderation
+
+import (
+	"database/sql"
+
+	"github.com/jonas747/discordgo"
+	"github.com/jonas747/dstate"
+	"github.com/jonas747/yagpdb/bot"
+	"github.com/jonas747/yagpdb/common"
+)
+
+type WarnRankEntry struct {
+	Rank      int    `json:"rank"`
+	UserID    int64  `json:"user_id"`
+	Username  string `json:"username"`
+	WarnCount int64  `json:"warn_count"`
+}
+
+func TopWarns(guildID int64, offset, limit int) ([]*WarnRankEntry, error) {
+	const query = `SELECT rank, warn_count, user_id FROM
+	(
+		SELECT RANK() OVER (ORDER BY count(message) DESC) AS rank, count(message) as warn_count, user_id

```

Comment:
why count(message) instead of count(*)?

Reply:
yeah, that was just random column name I used, will edit later to *